### ISSUE TITLE
Consider afterUpdate call after conntected

### ISSUE THREAD

**Issue by lukas-schardt** [webtides/element-js#10]
I think the initial rendering is kind of an update too and calling `afterUpdate` would make sense. Code that is needed in `afterUpdate` is most likely needed after the initial render too.

By calling `afterUpdate` after `connected`, we would not need to write duplicate code, needed in `afterUpdate` and `connected`. 

A solution could be to call the `afterUpdate` hook manually in our `BaseElement` after calling the `connected` hook.

**Comment by eddyloewen** [comment]
For some reason I just thought about this again. To still have the option to do something different on the first `afterUpdate` call, we could pass it an options object with a boolean flag to indicate the first update.

```javascript
afterUpdate({ firstUpdate }) {
  if (!firstUpdate) {
    //do something
  }
}
```

**Comment by eddyloewen** [comment]
Let's forget about the `{ firstUdpate }` option for now...
While implementing #60 I remembered why it was not so easy to do it right from the beginning...
The actual first `afterUpdate` would be called BEFORE the `connected` hook. That was the problem.
So everything will stay the same for now. But I have added a manual call to `afterUpdate` right after the `connected` hook just like @lukas-schardt suggested.
